tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -350,3 +354,33 @@ export const downloadResourceVersion = async (
   downloads.push(download)
   return downloads
 }
+
+export const emptyTrashBinResources = async (page): Promise<string> => {
+  await page.locator(emptyTrashBinButton).click()
+  await page.locator(actionConfirmationButton).click()
+  const message = await page.locator(notificationMessageDialog).textContent()
+  return message.trim().toLowerCase()
```

Comment:
I'm sorry I didn't write right away. here is also we can add response checking

Reply:
Fixed it in latest push